docs/UserGuide.md

```diff
@@ -87,7 +87,7 @@ Profile | me | category, descriptors
 
 ### Company
 
-Note: You must be viewing the **Companies** page in order to execute these commands.
+Note: You must be on the **Companies** page in order to execute these commands.
```

Comment:
for consistency maybe use tab because we usually describe them as tabs and not pages(?) for this and others below

Reply:
Yes, I agree that tabs actually make more sense in this context. Will update for all use cases